flask_bcrypt.py

```diff
@@ -56,13 +57,14 @@ def check_password_hash(pw_hash, password):
     is intended to be used as a helper function at the expense of the 
     configuration variable provided when passing back the app object. In other 
     words this shortcut does not make use of the app object at all.
-    
-    To this this function, simple import it from the module and use it in a 
-    similar fashion as the method would be used. Here is a quick example::
-        
+
+    To use this function, simple import it from the module and use it in a 
```

Comment:
I think this should also be “simply import”.

Reply:
You're right. I saw that, thought I fixed it too.